surveys/2024-annual-survey/translations/de.md

```diff
@@ -289,7 +289,7 @@ Type: select one
 - Ich weiß es nicht oder es interessiert mich nicht
 - Andere Antwort...
 
-### Welche nicht implementierten oder (nur nächtlichen) Funktionen möchten Sie stabilisieren?
+### Welche nicht implementierten (oder nur Nightly) Funktionen möchten Sie stabilisiert sehen?
```

Comment:
Not completely sure if the phrase "[...] stabilisiert sehen" actually exists in German, but the meaning is more correct probably. Before it said "which features do you want to stabilize?" (active instead of passive voice).

Reply:
wünschen?